jest.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    rootDir: ".",
+    testRegex: "tests/.*-(test)\\.jsx?$",
+    verbose: true,
+    bail: false,
+    collectCoverage: true,
+    collectCoverageFrom: [
+        "src/js/containers/**/*.{js,jsx}",
+        "src/js/redux/reducers/**/*.{js,jsx}",
+        "!node_modules/**",
```

Comment:
When running `npm test` I'm getting this output which I haven't seen before:
```Running coverage on untested files...  console.error node_modules/@fortawesome/react-fontawesome/index.js:325
    Could not find icon { prefix: 'fas', iconName: 'chart-pie' }

  console.error node_modules/@fortawesome/react-fontawesome/index.js:325
    Could not find icon { prefix: 'fas', iconName: 'chart-pie' }

  console.error node_modules/@fortawesome/react-fontawesome/index.js:325
    Could not find icon { prefix: 'fas', iconName: 'chart-pie' }

  console.error node_modules/@fortawesome/react-fontawesome/index.js:325
    Could not find icon { prefix: 'fas', iconName: 'chart-pie' }

  console.error node_modules/@fortawesome/react-fontawesome/index.js:325
    Could not find icon { prefix: 'fas', iconName: 'chart-pie' }
```
wondering if we need to mock the fontawesome library or something.

Reply:
@ebdabbs -- 

All the configuration is the same, but these errors don't appear in the `dev` branch.  Also, there are some other errors from tests being executed that points to async errors. I was able to fix some of them by adding `async/await` keywords to some of the tests.

All of this I think is likely attributable to the upgrade to `Jest`.

Reply:
Agreed, I think it falls out of the scope of this PR to address this!